### PR TITLE
`regrid_space` fix

### LINKED ISSUE #102: Fix remapping of NetCDF boundary condition files
`remap_space` is no longer needed for plotting, and it in fact creates problems, following the ClimaAtmos0.3.0 update.

### PR DESCRIPTION
**Pull request by LenkaNovak**
# PULL REQUEST

## Purpose and Content
This fixes element order issues alter ClimaAtmos0.3.0 update.

## Benefits and Risks
- b: produces correct spaces, enables further development of snow and pipeline PRs
- N/A

## Linked Issues
- Fixes #102 
- Closes #102 

## Tests
- land mask, initial ClimaAtmos Fields (unaltered by coupler) and solution ClimaAtmos Fields (altered by coupler) all produce the expected results. When testing, clear your environment - all temporary weightfiles should be deleted. 

Screen Shot 2022-07-26 at 11 58 21 PM

Screen Shot 2022-07-26 at 11 57 59 PM

## PR Checklist
- [x] This PR has a corresponding issue OR is linked to an SDI.
- [x] I have followed CliMA's codebase [contribution](https://clima.github.io/ClimateMachine.jl/latest/Contributing/) and [style](https://clima.github.io/ClimateMachine.jl/latest/DevDocs/CodeStyle/) guidelines OR N/A.
- [x] I have followed CliMA's [documentation policy](https://github.com/CliMA/policies/wiki/Documentation-Policy).
- [x] I have checked all issues and PRs and I certify that this PR does not duplicate an open PR.
- [x] I linted my code on my local machine prior to submission OR N/A.
- [x] Unit tests are included OR N/A.
- [x] Code used in an integration test OR N/A.
- [x] All tests ran successfully on my local machine OR N/A.
- [x] All classes, modules, and function contain docstrings OR N/A.
- [x] Documentation has been added/updated OR N/A.

### CODE REVIEW

File: experiments/AMIP/moist_mpi_earth/coupler_utils/unit_tester.jl [hunk 1]
```diff
@@ -27,40 +27,40 @@ domain = Domains.SphereDomain(FT(6731e3))
 mesh = Meshes.EquiangularCubedSphere(domain, 4)
 topology = Topologies.Topology2D(mesh)
 quad = Spaces.Quadratures.GLL{5}()
-boundary_space = Spaces.SpectralElementSpace2D(topology, quad)
+boundary_space_t = Spaces.SpectralElementSpace2D(topology, quad)
 
-land_mask = ClimaCore.Fields.zeros(boundary_space)
+land_mask_t = ClimaCore.Fields.zeros(boundary_space_t)
```
Comment (kmdeck) [inline experiments/AMIP/moist_mpi_earth/coupler_utils/unit_tester.jl:32]: Im curious about the _t subscript here :-)

**Comment by kmdeck** [comment]
Looks great Lenka. as far as I can tell this removes the regrid space and replaces it with boundary_space in each case. If this fixes the plotting issue it is good to go!

One minor fix - one of the regrid .g files slipped into the PR.

**Review by kmdeck — APPROVED** [review]

File: experiments/AMIP/moist_mpi_earth/coupler_utils/unit_tester.jl (hunk 1 above)
Comment (LenkaNovak) [reply experiments/AMIP/moist_mpi_earth/coupler_utils/unit_tester.jl:32]: It stands for test. 😇 I know this file only a temporary way of testing, but it was overwriting the variables from the driver, which made debugging a bit harder. :-P

**Comment by LenkaNovak** [comment]
> Looks great Lenka. as far as I can tell this removes the regrid space and replaces it with boundary_space in each case. If this fixes the plotting issue it is good to go!
> 
> One minor fix - one of the regrid .g files slipped into the PR.
Ooh! Nice catch! 😃

**Pull request merged**